patches/minecraft/net/minecraft/world/storage/MapDecoration.java.patch

```diff
@@ -1,17 +1,23 @@
 --- a/net/minecraft/world/storage/MapDecoration.java
 +++ b/net/minecraft/world/storage/MapDecoration.java
-@@ -83,6 +83,16 @@
+@@ -83,6 +83,22 @@
        return i;
     }
  
++   @Deprecated // Use overload with matrix stack and buffers. TODO 1.15 remove this
```

Comment:
Is this TODO supposed to say 1.16 and not 1.15?

Reply:
yes, thanks